### PR TITLE
Add design principles page

### PR DESCRIPTION
**Pull request by marcgurney**
cc @elianekabkab @fredtruman 

screen shot 2017-10-02 at 2 36 04 pm

### CODE REVIEW

File: client/components/ui-design-system/welcome/principles.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+      <h3>Inclusiveness</h3>
+      <p>Our products serve everyone from beginners learning how to use MongoDB for the first time to experts 
+      in the field. They empathize with the user, anticipate their goals and needs to deliver an experience 
+      that feels custom tailored. When a beginner feels stuck, the product provides guidance. When an expert 
+      desires a shortcut or advanced feature, the product supplies it exactly as the user would expect.</p>
+
+      <h3>Consistency</h3>
+      <p>We offer a full suite of products that fit into each stage of a user’s lifecycle and workflow. 
+      Information and interaction patterns that appear in different places are consistent visually and
```
Comment (fredtruman) [inline client/components/ui-design-system/welcome/principles.jsx]: > interaction patterns that appear in different places are consistent **visually and conceptually**.

🎊  Just generally a great statement.

File: client/components/ui-design-system/welcome/principles.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      Information and interaction patterns that appear in different places are consistent visually and 
+      conceptually. This allows users to move fluidly through our products, confident that what they learn in 
+      one will carry over to all others.</p>
+
+      <h3>Automation</h3>
+      <p>Our products convey an understanding that sometimes the best design is invisible to the user. While 
+      our products break complicated tasks into simple and intuitive flows, the biggest impact is achieved when 
+      the work is done for the user. Not just a matter of convenience, when we automate a task we reduce chance 
+      of user error, leading to a safer overall experience. Our products automate where possible while leaving 
+      control with the user where needed.</p>
```
Comment (fredtruman) [inline client/components/ui-design-system/welcome/principles.jsx]: > while leaving control with the user where needed.

... where needed [or desired].

?

File: client/components/ui-design-system/welcome/principles.jsx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      
+      <h3>Clarity</h3>
+      <p>Through the use of familiar patterns, guided workflows, inline instructions and contextual information, 
+      our products are understood without the need for a manual, where possible, and provide easy access to 
+      concise and clear documentation where needed. Terminology is carefully considered and maps closely to the 
+      purpose or concept it describes. Cognitive load is offloaded onto the application wherever possible.</p>
+
+      <h3>Integrity</h3>
+      <p>Information communicated to the user is accurate and thorough. Complete transparency instills 
+      confidence and trust, trust in the resilience and safety of the product, which is crucial for managing
```
[the remark this answers is not in the document]
Comment (marcgurney) [reply client/components/ui-design-system/welcome/principles.jsx]: Yeah, it should maybe be separated by an em dash. Or drop the comma and the double "trust", but then might be considered a run-on?

Either:
>Complete transparency instills confidence and trust—trust in the resilience and safety of the product—which is crucial for managing mission critical infrastructure.

Or
>Complete transparency instills confidence and trust in the resilience and safety of the product, which is crucial for managing mission critical infrastructure.

[the remark this answers is not in the document]
Comment (fredtruman) [reply client/components/ui-design-system/welcome/principles.jsx]: My grammar is not the best so I can't judge proper structure, but either of those options *read* more naturally to me.

[the remark this answers is not in the document]
Comment (elianekabkab) [reply client/components/ui-design-system/welcome/principles.jsx]: I like the second option, just simpler

**Pull request merged**